model/model.go

```diff
@@ -105,6 +118,7 @@ type Container struct {
 	MountSources  bool          `json:"mountSources" yaml:"mountSources"`
 	Command       []string      `json:"command" yaml:"command"`
 	Args          []string      `json:"args" yaml:"args"`
+	Lifecycle     *Lifecycle    `json:"lifecycle,omitempty" yaml:"lifecycle,omitempty"`
```

Comment:
Please consider updating model description in plugin-registry 
https://github.com/eclipse/che-plugin-registry#plugin-meta-yaml-structure
https://github.com/eclipse/che-plugin-registry/blob/master/build/scripts/meta.yaml.schema

Reply:
https://github.com/eclipse/che-plugin-registry/pull/442